extensions/wikia/UserLogin/js/UserSignup.js

```diff
@@ -30,47 +29,6 @@
 			this.initOptIn(this.wikiaForm);
 		},
 
-		loadCaptcha: function () {
-			if (this.useCaptcha) {
-				$.loadReCaptcha().fail(this.handleCaptchaLoadError.bind(this));
-			}
-		},
-
-		/**
-		 * Captcha is required for signup, so if it fails to load (possibly b/c google is blocked in China)
-		 * inform the user with a modal. Note, this is different from when a user fails the captcha test itself.
-		 */
-		handleCaptchaLoadError: function () {
-			require(['wikia.ui.factory'], function (uiFactory) {
-				$.when(uiFactory.init('modal'))
-					.then(this.createCaptchaLoadErrorModal.bind(this));
-			}.bind(this));
-
-			Wikia.Tracker.track({
-				action: Wikia.Tracker.ACTIONS.ERROR,
-				category: 'user-sign-up',
-				label: 'captcha-load-fail',
-				trackingMethod: 'both',
-				country: Wikia.geo.getCountryCode()
-			});
-		},
-
-		createCaptchaLoadErrorModal: function (uiModal) {
-			var modalConfig = {
-				vars: {
-					id: 'catchaLoadErrorModal',
-					classes: ['captcha-load-error-modal'],
-					size: 'medium',
-					title: $.msg('usersignup-page-captcha-load-fail-title'),
```

Comment:
Are these messages still needed for anything? If not, let's remove them from the i18n files and from the setup file that exports them to JS.